modules/culturefeed_agenda/src/Utility/TextProcessingTrait.php

```diff
@@ -0,0 +1,523 @@
+<?php
+
+namespace Drupal\culturefeed_agenda\Utility;
+
+use Drupal\Component\Utility\Unicode as CoreUnicode;
+
+/**
+ * Text processing trait.
+ */
+trait TextProcessingTrait {
+
+  /**
+   * Create a summary for the given text.
+   *
+   * @param string $text
+   *   The text to create a summary from.
+   * @param int $size
+   *   The size.
+   *
+   * @return bool|string
+   */
+  public function createSummary(string $text, int $size = 300) {
```

Comment:
Deze code is eigenlijk een copy / paste van de text_summary functie in drupal core. Best dit eruit, en vanaf nu rechtstreeks die gaan gebruiken

Reply:
Aha ja dat lijkt me zeker de beste optie om deze weg te halen en om te vormen naar Drupal core. Dit is bij deze gebeurd met text_format($text, NULL, $size)